src/app/pages/sharing/iscsi/extent/extent-form/extent-form.component.ts

```diff
@@ -28,7 +34,7 @@ export class ExtentFormComponent {
   protected fieldConfig: FieldConfig[] = [
     {
       type : 'input',
-      name : 'iscsi_target_extent_name',
+      name : 'name',
       placeholder : T('Extent name'),
       tooltip: T('Enter the extent name. The name cannot be an existing\
```

Comment:
`` Name of the extent. If <i>Extent size</i> is non-zero, this cannot be the name of file that already exists. ``